components/MintPage/MintNow.tsx

```diff
@@ -17,13 +21,54 @@ const MintNow = () => {
   const [containerRef, containerSizes] = useMeasure()
   const [timerRef, timerSizes] = useMeasure()
 
+  const [openModal, setOpenModal] = useState(false)
+
+  const { isConnected } = useAccount()
+
+  const [tierIQuantity, setTierIQuantity] = useState(0)
+  const [tierIIQuantity, setTierIIQuantity] = useState(0)
+  const [tierIIIQuantity, setTierIIIQuantity] = useState(0)
+
+  const increaseQuantity = (type: number) => {
+    switch (type) {
+      case 1:
+        setTierIQuantity(tierIQuantity + 1)
+        break
+      case 2:
+        setTierIIQuantity(tierIIQuantity + 1)
+        break
+      case 3:
+        setTierIIIQuantity(tierIIIQuantity + 1)
+        break
+      default:
+    }
+  }
+
+  const decreaseQuantity = (type: number) => {
+    switch (type) {
+      case 1:
+        if (!tierIQuantity) return
+        setTierIQuantity(tierIQuantity - 1)
+        break
+      case 2:
+        if (!tierIIQuantity) return
+        setTierIIQuantity(tierIIQuantity - 1)
+        break
+      case 3:
+        if (!tierIIIQuantity) return
+        setTierIIIQuantity(tierIIIQuantity - 1)
+        break
+      default:
+    }
+  }
+
   return (
     <SectionContainer>
       <div
-        className="relative w-full h-[100vh] flex justify-center items-start xl:items-center z-[1]"
+        className="relative w-full min-h-[100vh] flex justify-center items-start xl:items-center z-[1]"
         ref={containerRef}
       >
-        {canMintNow ? (
+        {!canMintNow ? (
```

Comment:
seems like you're `muting` this check, right?

might be best to `remove` this check, for now, because it's not being used 🚮 

If we need to add it back in, you can open a new PR.

what do you think?